test/stream/api/sink/statistics-clear-001-ok/006-statistics-clear/expected-response.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
That's too effective clear of statistics :smile: . It is better for check to see the zeros. Just resolve the comment.